activerecord/test/cases/test_case.rb

```diff
@@ -107,20 +107,12 @@ def clear_log; self.log = []; self.log_all = []; end
 
     clear_log
 
-    self.ignored_sql = [/^SAVEPOINT/, /^ROLLBACK TO SAVEPOINT/, /^RELEASE SAVEPOINT/]
```

Comment:
I've realized #36153 accidentally removed `/^BEGIN/`, and `/^COMMIT/`.
Adding "TRANSACTION" to the `/^SAVEPOINT/`, `/^RELEASE SAVEPOINT/`, `/^ROLLBACK/`, `/^BEGIN/`, and `/^COMMIT/` is fine to me.